content/features/extensibility/event-system/index.md

```diff
@@ -34,7 +113,230 @@ You can attach code to the following Container events:
 -	OnContainerPreRender
 -	OnContainerUnLoad
 
+For each type of Container Event Listener, there are a minimum of two parts:
+ 1. Registering the Container Event Listener
+ 2. Implementing the Container Event Listener
+
+The following example shows how to create an Event Listener for the OnContainer Load event. 
+
+Note: The steps to create the other Container Event Listeners are the same except your will change the ContainerEventType in the sample code below.
+
+To attach code to the Container Event, you must first register your Container Event Listener. In this example we will register our Container Event Listener when DNN starts loading for the first time. 
+
+There are several ways to accomplish this, the following example registers the events within a Web API route mapper since that code loads when DNN starts up.
+
+    using DotNetNuke.Application;
+    using DotNetNuke.UI.Containers.EventListeners;
+    using DotNetNuke.Web.Api;
+    
+    namespace SampleProject.DnnEvents.ContainerEvents
+    {
+        class RouteMapper : IServiceRouteMapper
+        {
+    
+            public void RegisterRoutes(IMapRoute mapRouteManager)
+            {
+    
+                // Add Your Web API Routes Here
+    
+                // Register Your Skin Events
+    
+                DotNetNukeContext.Current.ContainerEventListeners.Add(new DotNetNuke.UI.Containers.EventListeners.ContainerEventListener(ContainerEventType.OnContainerLoad, SampleContainerEventListener.OnContainerLoad));
+    
+    
+            }
+    
+        }
+    }
+
+
+In the above example, we are registering a Skin Event Listener called **SampleContainerEventListener.OnContainerLoad**. This is a static class and method that defines what code will run when the Container OnLoad event triggers. 
+
+Since all modules on a page in a DNN website are wrapped in a container, we can assume that this code will run when every container loads on every page within a DNN website.
```

Comment:
Does this run if the _Display Container_ tab-module setting isn't checked?

Reply:
I haven't tested but no I don't believe it does. These are triggered from the container base class, if the container isn't visible on the page then the container code should not trigger for that specific module/container combo.